CHANGELOG

```diff
@@ -1,7 +1,30 @@
+Version 5.0.0 (2023-04-04)
```

Comment:
Just for the sake of keeping full history, could we add version 4.1.2 changes here?